server/mvc/log.js

```diff
@@ -0,0 +1,4 @@
+module.exports.init = function(log){
```

Comment:
I'm working on a ([DevExamples](https://github.com/Starbow/website/blob/4d6890a263e94767860ce52b9c78ab41018e619d/server/mvc/models/DevExamples.js)) on a different branch. I'll add some logic explaining this behavior.